spec/lib/sufia/import/generic_file_translator_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Sufia::Import::GenericFileTranslator do
+  let(:sufia6_user) { "s6user" }
+  let(:sufia6_password) { "s6password" }
+  let(:translator) { described_class.new(sufia6_user: sufia6_user, sufia6_password: sufia6_password) }
+
+  let(:import_directory) { File.join(fixture_path, 'import') }
+  let(:gf_metadata) { JSON.parse(File.read(File.join(import_directory, 'generic_file_qr46r0963.json')), symbolize_names: true) }
+  let(:work_id) { 'qr46r0963' }
```

Comment:
This is slightly mysterious that is matches with the id in the fixture. May want a comment.